Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/RemoteMongoClientIntTests.swift

```diff
@@ -1078,8 +1078,8 @@ class RemoteMongoClientIntTests: BaseStitchIntTestCocoaTouch {
         sync.find(filter: ["hello": "computer"], options: nil, joiner.capture())
         guard let cursor = joiner.value(asType: MongoCursor<Document>.self),
             let actualDoc = cursor.next()else {
-            XCTFail("documents not found")
-            return
+                XCTFail("documents not found")
```

Comment:
[nit for whole file] I think the original indentation was correct, multi-line conditionals look weird. we can also just fix when we include linter